src/styles/zIndex.js

```diff
@@ -0,0 +1,10 @@
+export default {
+  menu: 10,
+  appBar: 11,
+  leftNav: 12,
+  overlay: 13,
+  dialog: 14,
+  layer: 20,
+  popover: 20,
+  tooltip: 21,
```

Comment:
> This is a big decision. because touching these later is breaking.

I agree, I have added this to the CHANGELOG.md
`zIndex, rework them to be more coherent (#2444)`

> about overlay. well dialog must be opened over leftnav, so it's overlay has to cover leftnav

Makes sense, I'm gonna add a property to the Overlay.